docs/codeql/codeql-cli/creating-and-working-with-codeql-packs.rst

```diff
@@ -34,24 +34,26 @@ You must specify:
 
 The ``codeql pack init`` command creates the directory structure and configuration files for a CodeQL pack. By default, the command creates a query pack. If you want to create a library pack, you must edit the ``qlpack.yml`` file to explicitly declare the file as a library pack by including the ``library:true`` property.
 
-Modifying an existing QL pack to create a CodeQL pack
------------------------------------------------------
+Modifying an existing legacy CodeQL pack to create a CodeQL pack
```

Comment:
Perhaps we should still refer to the old-style packs as "QL packs" to avoid confusion?

```suggestion
Modifying an existing legacy QL pack to create a CodeQL pack
```